examples/test_extract.py

```diff
@@ -1,35 +1,33 @@
-import krec
-import argparse
-from pathlib import Path
-
-
-"""
+"""Usage:
 # Just extract and view
 python examples/test_extract.py input.mkv -v
 
 # Extract and save
 python examples/test_extract.py input.mkv -o output.krec -v
 """
 
+import argparse
+import logging
+from pathlib import Path
+
+import krec
+
 
 def main(args):
-    print(f"Extracting from: {args.input_file}")
+    logging.info(f"Extracting from: {args.input_file}")
 
     try:
         extracted_krec = krec.extract_from_video(args.input_file, verbose=args.verbose)
-        print("Extraction successful")
-        print(f"Extracted KRec has {len(extracted_krec)} frames")
+        logging.info("Extraction successful")
+        logging.info(f"Extracted KRec has {len(extracted_krec)} frames")
 
         if args.output_file:
             output_path = Path(args.output_file)
             extracted_krec.save(str(output_path))
-            print(f"Saved to: {output_path}")
+            logging.info(f"Saved to: {output_path}")
 
     except Exception as e:
-        print(f"Error: {e}")
-        return 1
-
-    return 0
+        logging.error(f"Error: {e}")
```

Comment:
oh, i thought i disabled f-strings in logging in pylint... this is not ideal, see here: https://pylint.readthedocs.io/en/stable/user_guide/messages/warning/logging-fstring-interpolation.html